### PR TITLE
Migrate to webpack

### LINKED ISSUE #10: Migrate to webpack
Using https://github.com/phucbm/wellii

### PR DESCRIPTION
**Pull request by vuquangpham**
Migrate from `gulp` to `webpack`. We also updated some commands for deployment, such as:
```shell
# dev
npm run dev

# generate files for production
npm run prod
```

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 Phuc Bui
```
Comment (phucbm) [inline LICENSE]: Change copyright to ViiVue.

File: README.md [hunk 2]
```diff
@@ -104,14 +103,46 @@ leads to popup closing when that element clicked.
 
 ## Deployment
 
-Install gulp
+### Dev server
+
+Run production server
+
+```shell
+npm run web
```
Comment (phucbm) [inline README.md]: Do we still have this server?

File: README.md [hunk 3]
```diff
@@ -104,14 +103,46 @@ leads to popup closing when that element clicked.
...
+Generate UMD and module version
+
+```shell
+npm run prod
+```
+
+Generate UMD and module version then publish NPM package
+
+```shell
+npm run publish
```
Comment (phucbm) [inline README.md]: You can remove this command for now.

File: package.json [hunk 4]
```diff
@@ -1,13 +1,59 @@
...
+    "phucbm",
+    "javascript",
+    "es6"
+  ],
+  "main": "./dist/easy-popup.module.js",
+  "license": "MIT",
+  "scripts": {
+    "dev": "webpack serve --config config/webpack.dev.js",
+    "build": "cross-env ENTRY=dev webpack --config config/webpack.build.js",
+    "sass:compressed": "sass src/_style.scss:dist/easy-select.css --no-source-map",
```
Comment (phucbm) [inline package.json]: `easy-select`?

File: dev/index.html [hunk 5]
```diff
@@ -4,42 +4,12 @@
...
-        }
-        footer a {text-decoration:none; display:inline-block;}
-    </style>
 </head>
 <body>
 
 
 <div class="container">
-    <h1>Easy Popup v0.0.5</h1>
+    <h1 data-title="">Easy Popup v0.0.5</h1>
```
Comment (phucbm) [inline dev/index.html]: Just `Easy Popup` is enough. The correct version will be injected anyway.

**Review by phucbm — CHANGES_REQUESTED** [review]
### 1. Update README

For deployment section in README file, please change to this:

```shell
# Run dev server
npm run dev

# Build dev site
npm run build

# Generate production files
npm run prod
```

### 2. Consistent UI for open-source projects

Please install [honcau](https://www.npmjs.com/package/honcau) package for UI consistency throughout our open-source projects. You can use Easy Select as an example usage of this package. Don't forget to remove old CSS.

### 3. Push `dist` folder and test

You need to generate and push the `dist` folder onto GitHub.

Once you have the distribution files, please test them on some of our **dev projects** then send the list to me in a private message.

File: package.json (hunk 4 above)
Comment (vuquangpham) [reply package.json]: That's my fault. Sorry about that. I have changed it (more detail at https://github.com/viivue/easy-popup/pull/11/commits/ecaf7a369b834d48b432bfd23b5e893e196817a2), please check again! Thanks.

**Review by phucbm — APPROVED** [review]
LGTM.

**Pull request merged**